### ISSUE TITLE
Feature suggestion: use tidyselect in `initial_split()` for the strata argument

### ISSUE THREAD

**Issue by zenggyu** [tidymodels/rsample#84]
I think it would be nice if the `strata` argument of `initial_split()` can use tidyselect. It has the benefit of being more consistent with the tidyverse, and can also enable stratified sampling by multiple variables.

For example:

```
dataset %>%
    initial_split(strata = vars(strata_var1, strata_var2))
```

or:

```
dataset %>%
    initial_split(strata = vars(starts_with("strata")))
```

**Comment by macek2** [comment]
agreed. this would also address #94

**Comment by topepo** [comment]
See my response for #94. I'm going to change it to use `tidyselect` but for single columns only.

**Comment by topepo** [comment]
Closed by #100
